### PR TITLE
Add JavaScript Bangla Programming Resources

### PR DESCRIPTION
**Pull request by asifadib**
JavaScript Bangla Programming Youtube Tutorial Resources
Basic to Advance

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-bl.md [hunk 1]
```diff
@@ -103,7 +103,11 @@
 * [JavaScript Bangla Tutorial Course 2021](https://www.youtube.com/playlist?list=PLgH5QX0i9K3qzryglMjcyEktz4q7ySunX) - Anisul Islam
 * [JavaScript Behind The Scene Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDPaOrWvTIuhb5GRoJVWiE2) - Stack Learner
 * [JavaScript Full Playlist With Order](https://www.youtube.com/playlist?list=PL_XxuZqN0xVAJTV_1ZXwB1XIiFkK0ddZA) - Stack Learner
+* [JavaScript Bangla Tutorial Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9OkrURufHpGUUTBjJhO9Ghy) - Learn with Sumit
```
Comment (LuigiImVector) [inline books/free-programming-books-bl.md:106]:
```suggestion
* [JavaScript Bangla Tutorial Course 2021](https://www.youtube.com/playlist?list=PLgH5QX0i9K3qzryglMjcyEktz4q7ySunX) - Anisul Islam
* [JavaScript Bangla Tutorial Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9OkrURufHpGUUTBjJhO9Ghy) - Learn with Sumit
* [JavaScript Behind The Scene Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDPaOrWvTIuhb5GRoJVWiE2) - Stack Learner
* [JavaScript Full Playlist With Order](https://www.youtube.com/playlist?list=PL_XxuZqN0xVAJTV_1ZXwB1XIiFkK0ddZA) - Stack Learner
```

A little alphabetical error

**Comment by davorpa** [comment]
@asifadib sorry for conflicts. The file was renamed in #6021 to fix language locale `bl` > `bn`

Merge upstream will be enough

**Comment by davorpa** [comment]
> Thanks for solve the issues...

Not solved yet. This is your task.

Go to this branch `asifadib/patch-2` on your fork and merge `upstream/master`

---

**:writing_hand: TIP:** Making an upstream merge should be enough. Follow this steps in your local environment:
```shell
git fetch --all
git checkout master
git merge upstream/master
... take your time to resolve conflicts, if any ...
... ensure add changed files to stage area (`git add [filename]`) ...
git commit -m "Solve conflicts EbookFoundation/free-programming-books#5838"
git push origin
```

This `push` into your fork makes this PR to be synchronized automatically :wink:

**Comment by eshellman** [comment]
added  in #6435

**Pull request closed without merging**